catalog/lambda-policies.yaml

```diff
@@ -0,0 +1,12 @@
+- sid: "DenyLambdaWithoutVpc"
```

Comment:
@jamengual is this policy correct?

1. It says "Deny" but the value is `true` - should it be "false"?
2. Other policies use "Bool" in the test, not "Null"

```
condition:
    - test: "Bool"
      variable: "rds:StorageEncrypted"
      values:
        - false
```